Assignment-11-September-4/vite-project/src/Components/Login/ForgetPassword.tsx

```diff
@@ -0,0 +1,69 @@
+import { useForm, SubmitHandler } from "react-hook-form";
+import { FormControl, Button, Box, TextField, Typography } from "@mui/material";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useNavigate } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../../store/store";
+import { sendVerificationEmail } from "../../store/loginThroughCodeSlice";
+import { useEffect } from "react";
+import { sendVerificationEmailRequest } from "../../sagaStore/sagas/codeSagaSlice";
+
+type FormValues = {
+  email: string;
+};
+
+const ForgetPassword: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const { register, handleSubmit } = useForm<FormValues>();
+  const { status } = useAppSelector((state) => state.code);
+
+  useEffect(() => {
+    if (status === "success") {
+      toast.success("Details have been sent to your email");
+      navigate("/login");
+    } else if (status === "failed") {
+      toast.error("Failed to send details to your email");
+    }
+  }, [status]);
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    if (import.meta.env.VITE_MIDDLEWARE === "thunk") {
```

Comment:
make a settings file, put all env varibles thier and export them with constant varibles with default value.